### ISSUE TITLE
Helper position does not get recomputed when Helper dimensions change

### ISSUE THREAD

**Issue by LunaticPoring** [elrumordelaluz/reactour#412]
Hello there, 

We noticed a seemingly wrong behaviour with the helper positionning when its size changes between two steps.

**Reproduction scenario:**
https://codesandbox.io/s/xenodochial-mountain-m9c75?file=/src/App.js

**Analysis**
This seems to be related to the fact that `helperWidth `and `helperHeight ` in Popover.tsx only get recomputed on `window. 'resize' ` event

`  const { width: helperWidth, height: helperHeight } = useRect(helperRef)`

```
export function useRect<T extends Element>(
  ref: React.RefObject<T> | undefined,
  refresher?: any
): RectResult {
  const [dimensions, setdDimensions] = useState(initialState)
  const handleResize = useCallback(() => {
    if (!ref?.current) return
    setdDimensions(getRect(ref?.current))
  }, [ref?.current])

  useEffect(() => {
    handleResize()
    window.addEventListener('resize', handleResize)
    return () => window.removeEventListener('resize', handleResize)
  }, [ref?.current, refresher])

  return dimensions
}
```

(the reference to ref?.current doesn't change here since the div is still the same, hence not re-entering the useEffect function)

**Possible solution:** 
Add target dimensions to handleResize callback dependencies and add handleResize 

(not optimal since it would require to duplicate getRect code there)

**Comment by elrumordelaluz** [comment]
Should be solved in `@reactour/tour@2.3.1`. 

[Here](https://codesandbox.io/s/funny-wilson-9516u) the updated sandbo

![sandbox](https://user-images.githubusercontent.com/784056/145819399-2b9fde2d-3ecb-4d08-bd19-10ab2c876814.gif)
x

**Comment by LunaticPoring** [comment]
Awesome, thank you ! And what a reaction time 😮, took me longer to write this issue than for you to fix it 😁
